terraform/deploy/app/frontendservice.tf

```diff
@@ -0,0 +1,76 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# ECS Task Definition
+# ---------------------------------------------------------------------------------------------------------------------
+module "ecs-fargate-task-definition" {
+  source                       = "../../modules/fargate-task-definition"
+  name_prefix                  = var.name_prefix
+  container_name               = var.container_name
+  container_image              = local.container_image
+  container_port               = var.container_port
+  container_cpu                = var.container_cpu
+  container_memory             = var.container_memory
+  container_memory_reservation = var.container_memory_reservation
+  common_tags                  = local.common_tags
+  role_arn                     = "arn:aws:iam::${local.account[local.environment]}:role/${var.assume_role}"
+  account                      = lookup(local.account, local.environment)
+  log_configuration = {
+    secretOptions = []
+    logDriver     = "awslogs"
+    options = {
+      "awslogs-group"         = "/aws/ecs/${data.aws_ecs_cluster.ecs_main_cluster.cluster_name}/${var.name_prefix}"
+      "awslogs-region"        = var.region
+      "awslogs-stream-prefix" = "ecs"
+    }
+  }
+  environment = [
+    {
+      name  = "frontendService.user_container_task_definition"
+      value = "${var.name_prefix}-analytical-workspace"
+    },
+    {
+      name  = "frontendService.load_balancer_name"
+      value = "aws-analytical-env-lb"
+    },
+    {
+      name  = "frontendService.aws_region"
+      value = var.region
+    },
+    {
+      name  = "frontendService.user_container_url"
+      value = "aws-analytical-env.${local.root_dns_prefix[local.environment]}.${local.parent_domain_name[local.environment]}"
+    },
+    {
+      name  = "frontendService.user_container_port"
+      value = 8443
+    }
```

Comment:
These env vars are not needed for the frontend service, the env vars required are:
`REACT_APP_UI_LOGIN_URL` - Cognito hosted UI login page URL
`REACT_APP_API_CONNECT_ENDPOINT` - Orchestration service `/connect` endpoint URL

Reply:
Done 👍 
We have set a placeholder URL for now - the logic to generate this url doesn't exist yet. Will be implemented in another ticket.